Makefile

```diff
@@ -1,6 +1,7 @@
 SHELL := /bin/bash
 JAEGER_IMPORT_PATH = github.com/jaegertracing/jaeger
 STORAGE_PKGS = ./plugin/storage/integration/...
+GO = go
```

Comment:
What do you think about using `?=`? It means users can `export GO=gotip` just once then just `make test` each time during development, instead of needing to alway run `make test GO=gotip`.

It also offers the option of `make test GO=gotip` if users don't wish to "permanently" set their `GO` env var.

Reply:
I personally don't like such auto-import of env vars, who knows what else is in there...